pkgs/by-name/ro/rockbox-utility/package.nix

```diff
@@ -1,74 +1,56 @@
-{ lib
-, stdenv
-, fetchurl
-, cryptopp
-, libusb1
-, makeWrapper
-, pkg-config
-, qt5
-, withEspeak ? false, espeak ? null
+{
+  lib,
+  stdenv,
+  fetchgit,
+  cmake,
+  pkg-config,
+  qt5,
+  makeWrapper,
+  cryptopp,
+  libusb1,
+  espeak ? null,
+  withEspeak ? false,
 }:
 
-stdenv.mkDerivation  rec {
-  pname = "rockbox-utility";
-  version = "1.4.1";
+stdenv.mkDerivation rec {
+  pname = "rbutil";
```

Comment:
I was hoping this would fix `nixpkgs-update` which uses repology.org, where the package is called `rbutil`, but I don't know whether the package name or the attribute name prevails.

Reply:
Well, let's change it, then.